<a href="https://colab.research.google.com/github/Sobrero22/Python/blob/master/FeipeSobrero_Aula1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
print('mundo,mundo,vasto mundo')

mundo,mundo,vasto mundo


In [ ]:
help(print)

Help on built-in function print in module builtins:

print(...)
    print(value, ..., sep=' ', end='\n', file=sys.stdout, flush=False)
    
    Prints the values to a stream, or to sys.stdout by default.
    Optional keyword arguments:
    file:  a file-like object (stream); defaults to the current sys.stdout.
    sep:   string inserted between values, default a space.
    end:   string appended after the last value, default a newline.
    flush: whether to forcibly flush the stream.



Se você fizer uma corrida de 10 quilômetros em 43 minutos e 30 segundos, qual será seu tempo médio por milha? Qual é a sua velocidade média em milhas por hora?

In [ ]:
(43.5/60)/(10/1.61)

0.11672500000000001

In [ ]:
1/0.11672500000000001

8.567144998929106

Temos que o tempo médio por milha será: 0.12 h/mi e a velocidade média será 8.57 mi/h